### ISSUE TITLE
oc calibration app downloadlink does not work

### ISSUE THREAD

**Issue by Duller44** [iorodeo-streamlit/oc-calibration-app#1]
Hello, we tried to download the oc calibration app. The download for the app is not available via the following link.
https://iorodeo-streamlit-oc-calibration-a-oc-calibration-appapp-vy7s4o.streamlit.app/?ref=blog.iorodeo.com

**Comment by iorodeo-streamlit** [comment]
Yes it looks like there was an issue with the poetry.lock file.  It should be running again.
